unity3d/Assets/Swrve/SwrveSDK/SwrveSDK.cs

```diff
@@ -1323,20 +1323,20 @@ public string GetAppStoreLink (int gameId)
     /// <returns>
     /// In-app message for the given event.
     /// </returns>
-	public SwrveMessage GetMessageForEvent (string eventName) {
-		if (!checkCampaignRules (eventName, SwrveHelper.GetNow())) {
-			return null;
-		}
+	  public SwrveMessage GetMessageForEvent (string eventName, IDictionary<string, string> payload=null) {
```

Comment:
It's not changing the signature, it's adding a new, default argument.  People can still call it about `GetMessageForEvent("asdf")`, but they (and we) can also call `GetMessageForEvent("asdf", new Dictionary<string, string>{ ... })`.